smrf/utils/utils.py

```diff
@@ -39,15 +33,34 @@ def type_func(self, value):
         Attempt to convert all the values to upper case.
 
         Args:
-            value: A single string in a a config entry representing a
-                station name
+            value: A single string in config entry representing a station name
         Returns:
             value: A single station name all upper case
         """
 
         return value.upper()
 
 
+class CheckRawString(CheckType):
+    """
+    Custom `inicheck` checker that will not change the input string
+    """
+
+    def __init__(self, **kwargs):
+        super(CheckRawString, self).__init__(**kwargs)
+
+    def type_func(self, value):
+        """
+        Do not change the passed value at all
+        Args:
+            value: A single string
+        Returns:
+            value: A single string unchanged
+        """
+
+        return value
```

Comment:
Think this would be worth opening a ticket for and move into 'inicheck'

Reply:
Agreed.

https://github.com/USDA-ARS-NWRC/inicheck/issues/55

Reply:
USDA-ARS-NWRC/inicheck/issues/54

Reply:
Now we can flip a coin :-)

Reply:
You got a lower number so you won!